examples/check-if-opened-with-testcafe/README.MD

```diff
@@ -0,0 +1,17 @@
+# Check Whether Web Page Is Opened With TestCafe
```

Comment:
a very minor thing: I guess we shouldn't capitalize `Is` in titles, should we?

Reply:
I'm pretty sure we capitalize verbs. [Source](https://capitalizemytitle.com/#:~:text=The%20rules%20are%20fairly%20standard,%2C%20coordinating%20conjunctions%2C%20and%20prepositions.)

Reply:
```suggestion
# Check whether the web page is opened with TestCafe
```